recipes/perl-xml-parser/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.44_01" %}
+
+package:
+  name: perl-xml-parser
+  version: {{ version }}
+
+source:
+  fn: XML-Parser-{{ version }}.tar.gz
+  url: https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-{{ version }}.tar.gz
+  sha256: 5310ea5c8c707f387589bba8934ab9112463a452f828adf2755792d968b9ac7e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl 5.22.2.1
+    - expat
+  run:
+    - perl 5.22.2.1
+    - expat
+
+test:
+  imports:
+    - XML::Parser
```

Comment:
These are not Python imports. I believe we need to create a perl script to test that.